### ISSUE TITLE
Get child pages by parent page id

### ISSUE THREAD

**Issue by vzabauski** [brianbunke/ConfluencePS#6]
My Confluence structure looks like this:
![untitled diagram](https://cloud.githubusercontent.com/assets/17778043/20150032/da6b1da8-a6c4-11e6-9c1f-e850f9cfe99f.png)

I can't refer to my parent page by name because of API restrictions (too many pages), so I refer to it by its ID and I get only it's name, id etc.
Looks like everything works how it's supposed to but I'd like to know, is there any way (using your module) to get all child pages from a parent one?

If not, I'll try to write a function that will do it using [this ](https://muddytm.github.io/blag/2016/03/24/Confluence-API-Getting-children-of-a-page.html)example.

**Comment by brianbunke** [comment]
No method right now. It's available in the API, so I'm calling this a feature request and will get around to implementing it. :smile: 

Also, just to clarify your API restriction comment, two things are possible right now to help circumvent the limit:

1. `Get-WikiPage -Title 'confluence doc' -Limit 500`
    1. By ignoring -Expand, you can at least increase your page title search from 100 to 500 pages
2. `Get-WikiPage -Title 'confluence doc' -Limit 500 -SpaceKey ITKB`
    1. By adding a space key, that filter is observed first
    2. You will get the first 500 results only within the correct space

**Comment by vzabauski** [comment]
I used this workaround. 😄 
```
$all_pages = Get-WikiSpace -Name "KB" | Get-WikiPage -Limit 1000
$pages1 = Get-WikiSpace -Name "KB" | Get-WikiPage -Limit 1000 -Expand
$c = Compare-Object $all_pages $pages1 -PassThru
$pages2 = $c | Get-WikiPage -Expand
$pages = $pages1 + $pages2
```
It doesn't consume much time since I don't have much pages that doesn't fit into API limit with `-Expanded` option. But it's not good for large amounts of pages.

**Comment by vzabauski** [comment]
I wrote a function that gets child pages by parent id. Sadly, I couldn't get -Expand option working.
```
http://example.com/rest/api/content/1234/child?expand=page.body.VIEW
http://example.com/rest/api/content/1234/child?expand=page&start=20&limit=10
```
Examples from Confluence are not working for me, I'm getting empty respond.

**Comment by ScottWelker** [comment]
Strange. In the v3.1 REST API an added option (might have been expand) caused the result to include child pages. Does the v5.

**Comment by vzabauski** [comment]
Can you give me an example or a link?

**Comment by ScottWelker** [comment]
Sure vzabauski. I am raveling at the moment vzabauski. 

I'll endeavor to dig something up. However, I haven't used the v5.x REST API except to create pages via brianbunke's GREAT ConfluencePS 'wrapper'. 

Confluence continually disappoints so I would not be too shocked if the option (include children) is removed in the later APIs. I'll see what I can find.

**Comment by brianbunke** [comment]
`Get-WikiChildPage` introduced. Issue resolved in #7 and supplemented in commit 359908b. 🎉
